02-source_channel/010-serviceaccount.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
```

Comment:
just curios, why you need this?

Reply:
@slinkydeveloper, b/c the `k8s` source needs that